src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java

```diff
@@ -45,11 +57,11 @@ protected Table findTable(Schema schema, String dbName, String tableName) throws
 
 		MysqlParserListener listener = new MysqlParserListener(currentDB);
 
-		System.out.println("Running parse on " + sql);
+		LOGGER.debug("SQL_PARSE <- \"" + sql + "\"");
 		ParseTree tree = parser.parse();
 
 		ParseTreeWalker.DEFAULT.walk(listener, tree);
-		System.out.println(tree.toStringTree(parser));
+		LOGGER.debug("SQL_PARSE ->   " + tree.toStringTree(parser));
 
```

Comment:
get an input stream, get a lexer on the input stream, get a token stream from lexer, get a parser on the token stream, get a listener on the DB, then parse into a tree, then walk the tree with the listener. 

What a function! Don't have any suggestions though. :D This code has to go somewhere.